CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ Breaking changes result in a different major. UI changes that might break custom
 
 ## [next-version]
 
+## [0.15.0]
+
+### Changed
+- Internal: Use HTTP protocol to post documents to the server instead of websockets
+
+### Fixed
+- Public: Websockets connection issues
```

Comment:
I think in this one we can drop websockets, since most clients will not know that the protocol is the cause. something like "Fixed intermittent connection problem"